README.md

```diff
@@ -21,17 +28,11 @@ helm repo add camunda https://helm.camunda.io
 helm repo update
 ```
 
-The charts are represented in the following image:
-![HELM CHARTS](imgs/HelmChartImage.png)
-
+## Components
```

Comment:
:thought_balloon: Maybe we move that under the overview section ?

Reply:
just to make sure I got it right :D

you mean removing `components` section and keeping the content under `overview` section
or
making `components` as a sub-section under `overview`?

I'm in favor of the first one, but both are fine to me.

Reply:
Yeah I was thinking of making it of a sub-section, but you can also remove the section again then it is similar as before :sweat_smile: